jupyter/datascience/ubi9-python-3.11/Pipfile

```diff
@@ -7,44 +7,50 @@ name = "pypi"
 
 [packages]
 # Datascience and useful extensions
-boto3 = "~=1.34.50"
+boto3 = "~=1.35.5"
```

Comment:
crazy, now 1.35.8 is the latest; they release this all the time! https://pypi.org/project/boto3/

Reply:
true, boto3 do release almost every day!